runtime/jcl/common/system.c

```diff
@@ -385,6 +385,17 @@ jstring getEncoding(JNIEnv *env, jint encodingType)
 
 		case 1: 		/* platform encoding */
 			encoding = getPlatformFileEncoding(env, property, sizeof(property), encodingType);
+			{
+				UDATA handle = 0;
+				PORT_ACCESS_FROM_ENV(env);
+				if (0 == j9sl_open_shared_library("java", &handle, J9PORT_SLOPEN_DECORATE)) {
+					void (*nativeFuncAddrJNU)(JNIEnv *env, const char *str) = NULL;
+					if (0 == j9sl_lookup_name(handle, "JNU_InitializeEncoding", (UDATA*) &nativeFuncAddrJNU, "VLL")) {
```

Comment:
`"VLL"` doesn't match the `JNIEnv *env, const char *str` declaration.  Is this correct?

Reply:
As per `omrsl_lookup_name` API doc, `V` is for `void` (return type in this case), `L` is for `object / pointer (32 or 64, depending on platform)` (parameter types in this case). Some existing code use same signature such as following code within `stdinit.c`.
```
jstring (JNICALL *nativeFuncAddr)(JNIEnv *env, const char *str) = NULL;
j9sl_lookup_name(javaLibHandle->handle, "JNU_NewStringPlatform", (UDATA*)&nativeFuncAddr, "LLL");
```
i.e., `LLL` for `jstring (JNIEnv *env, const char *str)`.

Additional note: this signature parameter seems only used at `AIX J9OS_I5` platform (Window only uses it for calculation of padding length). It is not straightforward to do a quick verification.

Reply:
Thanks for checking @JasonFengJ9.  I was expecting a "P" or similar character for pointers.

Reply:
As per `omrsl_lookup_name` API doc, `P` is for `pointer-width platform data. (in J9 terms an intptr_t)`. I followed the existing code and chose `L` in this case. Probably need run a test on an `AIX J9OS_I5` machine to determine if there is any difference between `L` and `P` for the sigatures.

Reply:
Apparently there is no `AIX i` series machines available in the farm.